.github/workflows/build-native.yml

```diff
@@ -6,7 +6,7 @@ jobs:
   Build-Native-Images:
     strategy:
       matrix:
-        os: ["ubuntu-latest", "macos-10.15"]
+        os: ["ubuntu-latest", "macos-11.0"]
```

Comment:
As it turns out `ubuntu-latest` is too new for many linux users (Redhat).  These need to build on `18.04`, which will soon expire.

Reply:
Switched to ubuntu-18.04 and switched back to macos-10.15.